Modules/_datetimemodule.c

```diff
@@ -2986,6 +2986,61 @@ date_fromisoformat(PyObject *cls, PyObject *dtstr)
     return NULL;
 }
 
+
+static int _iso_long_year_helper(int year) {
```

Comment:
For functions, the opening brace needs to be on its own line. See [PEP 7](https://www.python.org/dev/peps/pep-0007/).

Reply:
(The return type also needs to be on its own line.)

Reply:
Thanks for the review, good catch. I ended up removing this function in the end so it is moot now, but I did move the `{` to its own line in the `date_fromisocalendar`.